external-crates/move/Cargo.toml

```diff
@@ -155,6 +158,23 @@ move-vm-test-utils = { path = "crates/move-vm-test-utils" }
 move-vm-types = { path = "crates/move-vm-types" }
 prover_bytecode = { path = "crates/move-stackless-bytecode", package="move-stackless-bytecode" }
 
+# v0
```

Comment:
You don't want to make these workspace deps. Instead, we should follow the pattern that we use for other versioned crates, which is to update the toml files of the packages that depend on them to refer to them at their versioned path, but their unversioned name.

The reason for this is that it's much easier to automate (the script doesn't need to look at rust source files and find all references to the latest version and rename that to the versioned form).

See for example:

https://github.com/MystenLabs/sui/blob/127f2cb31699ebddbe1ea4935d264a235ca2e9ca/external-crates/move/move-execution/v1/crates/move-vm-runtime/Cargo.toml#L23